db/schema.rb

```diff
@@ -176,6 +176,7 @@
     t.datetime "updated_at", null: false
     t.integer "min"
     t.integer "max"
+    t.datetime "vssc_extract_date"
```

Comment:
For what it's worth, I don't know that `vssc` adds anything to this column name. Unless this is documenting the original source of the date. I'm not even sure I know what VSSC is, and I'm more likely than most to know that information!